docs/framework-agnostic-plugins.md

```
@@ -0,0 +1,2581 @@
+  workspace?: {
+    get: () => Workspace | null;
+    create: (name: string, type?: string) => Workspace;
+    addElement: (resultUuid: string, settings?: Record<string, unknown>) => WorkspaceElement;
+    removeElement: (elementId: string) => void;
+    updateElement: (elementId: string, updates: Partial<WorkspaceElement>) => void;
+    reorderElements: (elementIds: string[]) => void;
+  };
+}
+```
+
+**使用例: プレゼン作成ワークフロー**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)